### PR TITLE
508 imagelib search php

### PR DESCRIPTION
**Pull request by NikitaSalikov**
Added Doctype and primary language at the top
Lines 11-22 changed FILTER to htmlspecialchar Created labels for taxontype( 104), phuid (122), tags(139), imagecount(167), imagetype(182), 

Additionally:

Content/imagelib/search.en.php - added strings in English to labeled values
Content/imagelib/search.es.php - added strings in Spanish to labeled values (Not sure if correct)
Content/imagelib/search.fr.php - added strings in French to labeled values(Not sure if correct)
classes/ImageLibrarySearch - lines 393 , 422, 430 changed FILTER to htmlspecialchar classes/OccurenceTaxaManager - line 379 changed FILTER to htmlspecialchar.

Pull Request Checklist:

- [X] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [N/A] Hotfixes should be merged into both the `Development` and `master` branches (at the same time).
- [X] All new text is preferrably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages)
- [X] There are no linter errors
- [N/A] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [X] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [N/A] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [X] Comment which GitHub issue(s), if any does this PR address
- [X] It is the code author's responsibility to merge their own pull request after it has been approved
- [X] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [N/A] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [X] If the dev team has agreed that this PR represents the last PR going into the Development branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place

Thanks for contributing and keeping it clean!

### CODE REVIEW

File: imagelib/search.php [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<!DOCTYPE html>
+
+<html lang="<?php echo $LANG_TAG ?>">
```
Comment (Atticus29) [inline imagelib/search.php]: There's an <html> tag in ln. ~43. It would be better to add the lang attribute there instead and remove this, I think.

**Review by Atticus29 — CHANGES_REQUESTED** [review]
All of the current changes look great! Thank you for helping with all of that!

I think there are a few more to do, though:

Screen Shot 2023-07-17 at 10 40 14 AM

I think we should maybe add a "Taxon" label to this input:
Screen Shot 2023-07-17 at 12 58 01 PM


These should be resolved:

Screen Shot 2023-07-17 at 12 59 40 PM
Screen Shot 2023-07-17 at 1 00 48 PM

Screen Shot 2023-07-17 at 1 10 03 PM

Finally, there's the onChange issue here:

Screen Shot 2023-07-17 at 1 15 17 PM

I'd recommend implementing an onChange and an onKeypress instead of just the onChange. I believe that I implemented this in a different PR. If I forget to follow up and post that in a comment here, please ask me about it!

**Comment by NikitaSalikov** [comment]
Hey! 

Ready for review. Here is what I did:

Line 43: Moved Lang attribute from top
Line 112, 116-117: Added Labels to input “taxa”, “usethes” and LANG lines “Search by” and “Include Synonyms” in lang/imagelb/search.* (en/es/fr) in respective languages.
Line 123,140,183: Added id for select

(!)Line179: Added onKeypress. Not sure if implementation is correct.

**Review by Atticus29 — COMMENTED** [review]
Nice job, @NikitaSalikov !

I re-ran TotalValidatorPro, and it seems happy with the changes except for for onchange/onkeypress for the `<select>` element.

It also stil wants you to either add text to `<meta name='keywords' content='' />` or to remove the `content=''` attribute.

In terms of the onchange listener, I think that our solution needs to be a little more thorough. I believe that this warrants a separate ticket, so don't worry about fixing that for this current ticket. If you fix the `content=''` issue, I'll approve this PR.

**Review by Atticus29 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**